app/controllers/api/v1/groups_controller.rb

```diff
@@ -0,0 +1,44 @@
+class Api::V1::GroupsController < ApplicationController
+  def index
+    @groups = Group.all
+
+    render json: @groups
+  end
+
+  def show
+    @group = Group.find(params[:id])
```

Comment:
Yeah, after i comment here, i searched about and i read this two blog posts, and now i agree about the way that you did. It's better the code seems obvious, and with this before action in some cases can be confused.